### ISSUE TITLE
Support for free style workouts

### ISSUE THREAD

**Issue by MakeGnomeWebFast** [wger-project/wger#1066]
## Use case
Users currently cannot perform a workout that is not pre-planned.

## Proposal
A new workout button to add and perform exercises as the training goes would have been great, especially when the workout is not part of a plan.

**Comment by rolandgeider** [comment]
Yes, this is definitely something that would be really useful. I'm setting the 2.2 milestone, where the plan is making the workouts more flexible
